dug_4_osfv.md

```diff
@@ -0,0 +1,361 @@
+class: center, middle, intro
+
+# &#x1F386; Dasharo Open Source Firmware Validation &#x1F386;
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# Agenda
+
+* Introduction to Dasharo Open Source Firmware Validation (OSFV)
+* Exploring OSFV v0.2
+* Recent improvements
+* Getting started with OSFV
+* Upcoming Features: OSFV v0.3 and Beyond
+* Q&A
+
+???
+
+---
+
+# History
+
+.center.image-99[![](/img/osfv.png)]
+
+* We've been using OSFV at least since 2018, when we've started validating PC
+  Engines coreboot releases on a monthly basis
+* Using those scripts we've executed over **50k** tests publicly releasing **150+**
+  binaries based on open-source firmware
+* Initially, it was closed source because of assumption that validation provides
+  majority of the value in open-source firmware development
+
+---
+
+# Dasharo OSVF
+
+.center[https://github.com/Dasharo/open-source-firmware-validation]
+
+* Since then, these scripts came through many iterations, supporting more
+  different products
+* At some pint we have decided to open-source what we have and start
+  maintaining and iproving it as an open-source product
```

Comment:
spellchecker

Reply:
ehh too bad codespell does not catch that... I would really love to not use anything than pre-commit for maintainting quality

Reply:
Let's see https://github.com/codespell-project/codespell/pull/3197